monai/transforms/transforms.py

```diff
@@ -175,11 +219,12 @@ def __call__(self, filename):
             img = correct_nifti_header_if_necessary(img)
             header = dict(img.header)
             header['filename_or_obj'] = name
-            header['original_affine'] = img.affine
+            # header['original_affine'] = img.affine
```

Comment:
I think maybe setting original_affine at loader is more clear.
Otherwise, remove commented line if we don't need to use it anymore.

Reply:
sure, now fixed, if we don't change the affine, then we don't create `original_affine`